src/cascadia/UIHelpers/Converters.cpp

```diff
@@ -105,4 +105,35 @@ namespace winrt::Microsoft::Terminal::UI::implementation
 
         return maxVal;
     }
+
+    double Converters::PaddingValueFromIndex(const winrt::hstring& paddingString, uint32_t paddingIndex)
```

Comment:
Maybe this should also take an `enum` parameter instead of index but this is only called from XAML. Maybe it is okay?